yaml/src/main/java/com/fasterxml/jackson/dataformat/yaml/YAMLGenerator.java

```diff
@@ -116,6 +116,15 @@
          * @since 2.9
          */
         INDENT_ARRAYS(false),
+        /**
+         * Feature enabling of which adds indentation with indicator for array entry generation
+         * (default indentation being 2 spaces).
+         *<p>
+         * Default value is `false` for backwards compatibility
+         *
+         * @since 2.11.3
```

Comment:
As per not on PR itself, new features need to be added in a minor version, not patch (as per SemVer guidance)

Reply:
updated to 2.12.0